### DISCUSSION TITLE
Recommended workflow for code development

### DISCUSSION THREAD

**Discussion by msorens** [microsoft/vscode-discussions#538] [Extension Development QnA] [answered]
Now that I have made my VS Code extension publicly available on the VS Code Marketplace (yay!) I find my development workflow in a quandary.
Previously, when I am preparing a pull request I would manually create a VSIX bundle, provide that to reviewers, they would install from the command line, and everything works as expected.

Now that I have release `v1.0.1` on the marketplace, I tried creating a new bundle for `v1.0.1-next-B`. The problem is it appears to install, but the changes in the readme do NOT appear on my extension's page within VS Code.

<img width="766" alt="image" src="https://user-images.githubusercontent.com/6817500/227367460-9e33023e-886e-4f08-8a61-0b75c6251b8a.png">

So what is the proper incantation to be able to do this?

Update: 

Re-reading the fine print on the docs, it looks like I must increment the semver to get it to load properly. So I went with `v1.0.2-next` as my temporary, in-house version. 

Still interested to find out what other folks do for your workflow.

**Comment by gjsjohnmurray** [comment, marked as answer]
Does the problem still happen if you version your VSIX as v1.0.**2**-next-B instead? I think doing this will also get rid of the "Update to v1.0.1" offer.

See https://github.com/microsoft/vscode/issues/108584

**Comment by msorens** [comment]
@gjsjohnmurray 👍 Right; that is what I ended up doing, if you saw the last couple lines of my original posting.

**Reply by gjsjohnmurray** [reply]
I see your update now. I posted after reading the original notification email. Glad you found the solution.
